client/src/reducers/transforms/changeDatatype.js

```diff
@@ -6,7 +6,7 @@ function makeParser(parser) {
   return (rows, idx, onError, args) => {
     const deleteMarker = {};
     const { defaultValue } = args;
-    const dv = defaultValue != null ? parser(defaultValue) : null;
+    const dv = defaultValue != null ? defaultValue : null;
```

Comment:
I'll change this to just let the default value flow through.. I missed this in the refactor. Good catch!